BodyPart Nose
BodyPart Neck
BodyPart RShoulder
BodyPart RElbow
BodyPart RWrist
BodyPart LShoulder
BodyPart LElbow
BodyPart LWrist
BodyPart RHip
BodyPart RKnee
BodyPart RAnkle
BodyPart LHip
BodyPart LKnee
BodyPart LAnkle
BodyPart REye
BodyPart LEye
BodyPart REar
BodyPart LEar
